shell/shared/admin.js

```diff
@@ -80,8 +114,33 @@ Accounts.identityServices.saml = {
     return serviceEnabled("saml") && globalDb.isFeatureKeyValid();
   },
 
+  getLoginId: function (identity) {
+    return user.services.saml.id;
+  },
+
+  initiateLogin: function (loginId) {
+    Meteor.loginWithSaml();
```

Comment:
Turns out `loginWithSaml()` currently requires the parameter `{provider:"default"}`, so this doesn't work. Fixing...